docroot/themes/custom/yukonca_glider/patterns/molecules/callout_card/pattern-callout-card.html.twig

```diff
@@ -2,24 +2,27 @@
 {% set classes = [
   prefix,
   prefix ~ '--' ~ variant,
+  'bg-alert-background',
 ] %}
 
 {% set attributes = attributes.addClass(classes).setAttribute('data-aos', 'fade-up').setAttribute('data-aos-once', 'true') %}
```

Comment:
Why don't we use this variable in the component?

Reply:
Because this variable is used in section pattern (in the bottom of this file)